src/Symfony/Component/Notifier/Bridge/Smsc/SmscTransport.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We don't use strict types on this repository.

Reply:
Got it, will fix now

Reply:
Can abbot help us spot those things in the future?